python-squid/src/test/resources/docstring.py

```diff
@@ -0,0 +1,33 @@
+"""
+This is a module docstring
+"""
+
+def function_with_docstring():
+	"""This is a function docstring"""
+	pass
+
+def function_without_docstring():
+#   ^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
@pynicolas It was a blind copy from the test file for a Check class. Removed. Thanks!